stable/mlrun/values.yaml

```
@@ -166,9 +169,21 @@ ui:
   # runAsNonRoot: true
   # runAsUser: 1000
 
-rbac:
+service:
+  create: true
+
+deployment:
   create: true
 
+rbac:
+  clusterResources:
+    create: false
+  namespaced:
```

```suggestion
  namespacedResources:
```
namespaced alone is not very clear